mmv1/third_party/terraform/resources/resource_apigee_env_keystore_alias_pkcs12.go

```diff
@@ -158,7 +144,7 @@ Flag is set to Yes if the certificate is valid, No if expired, or Not yet if not
 			},
 			"password": {
 				Type:        schema.TypeString,
-				Optional:    true,
```

Comment:
computed and forcenew can be set at the same time. Consider:
1. if the field is output-only: only Computed:true
2. if the field is a input field + immutable: Optional: true + ForceNew: true
3. if the field is a input field + updatable: Optional: true + adding a update function for the resource